iocore/net/UnixNetAccept.cc

```diff
@@ -443,19 +430,28 @@ NetAccept::acceptFastEvent(int event, void *ep)
       goto Lerror;
     }
 
+    vc = (UnixNetVConnection *)this->getNetProcessor()->allocate_vc(e->ethread);
+    if (!vc) {
+      goto Ldone;
```

Comment:
Might as well do con.close() here.  Might not be needed now, but might close a leak in the future.  This is the error case anyway, shouldn't micro-optimize here.